lib/open_telemetry/honeycomb/config.ex

```diff
@@ -0,0 +1,115 @@
+defmodule OpenTelemetry.Honeycomb.Config.AttributeMap do
```

Comment:
Do the configs really need to be structs?

Reply:
You could ask the same of any choice. What do you prefer, instead, and why?